tests/integration_tests/scheduler/test_generic_driver.py

```diff
@@ -127,7 +128,7 @@ async def test_kill_actually_kills(driver: Driver, tmp_path, pytestconfig):
         job_kill_window = 1
```

Comment:
I think I have seen flakiness also when --openpbs and --lfs is not set as the pytest option. Then one needs to bump these values as well, but you may just mark the test as flaky and say a couple of reruns is ok.